woocommerce/woocommerce-ios
# Update to shared lanes

This PR removes the local `ios_update_metadata_source.rb` and updates the `release-toolkit` to use the shared one (the related PR is [here](https://github.com/wordpress-mobile/release-toolkit/pull/109)).

**To test:**
- Checkout a new branch from this one.
- Push it to GitHub and setup tracking.
- Run `bundle exec fastlane update_appstore_strings version:3.9`
- Verify that `AppStoreStrings.pot` is updated and that the lane finishes without errors.
- Delete the test branch from GitHub and your local repository.

Update release notes:
